libs/novui/src/types/LocalizedMessage.ts

```diff
@@ -0,0 +1,13 @@
+/**
```

Comment:
Can you elaborate on how this definition will help us with i18 in the future? Is it like a marker for places that need to be replaced with i18n-aware strings?

I'd also argue whether it belongs to the design system package or the web app. 

If that's the case, I'd suggest instead of using that to use a `t("this is a string")` function, that lives outside the design system that currently is just a unit function `f(x) => x`/

Reply:
@antonjoel82 , similarly, can you please respond in this thread and we'll continue the conversation here. A subsequent PR can be used to modify this proposal if necessary. @SokratisVidros , is that agreeable?

Reply:
Of course!